manage.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+"""Django's command-line utility for administrative tasks."""
+import os
+import sys
+
+
+def main():
+    """Run administrative tasks."""
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'my_project.settings')
+    try:
+        from django.core.management import execute_from_command_line
+    except ImportError as exc:
+        raise ImportError(
+            "Couldn't import Django. Are you sure it's installed and "
+            "available on your PYTHONPATH environment variable? Did you "
```

Comment:
Hi Sara. Good job so far with the project setup. I tied to run the server but got these errors:
![image](https://github.com/enocol/college_market_place/assets/129378066/52f4d517-ea80-41ed-abe7-eed5e0987d47)

I think we still need to have a look and see whats causing these errors and fix them before we merge. Could you have a second look and see if you can fix the errors. The errors show up when you run <python manage.py runserver>